dev/wlp-jakartaee-transform/rules/jakarta-renames.properties

```diff
@@ -79,3 +79,18 @@ javax.xml.bind.attachment=jakarta.xml.bind.attachment
 
 javax.transaction=jakarta.transaction
 
+javax.validation=jakarta.validation
+javax.validation.bootstrap=jakarta.validation.bootstrap
+javax.validation.constraints=jakarta.validation.constraints
+javax.validation.constraintvalidation=jakarta.validation.constraintvalidation
+javax.validation.executable=jakarta.validation.executable
+javax.validation.groups=jakarta.validation.groups
+javax.validation.metadata=jakarta.validation.metadata
+javax.validation.spi=jakarta.validation.spi
+javax.validation.valueextraction=jakarta.validation.valueextraction
+org.hibernate.validator=org.hibernate.validator
+org.hibernate.validator.cdi.internal=org.hibernate.validator.cdi.internal
+org.hibernate.validator.cdi=org.hibernate.validator.cdi
+org.hibernate.validator.internal.engine.valueextraction=org.hibernate.validator.internal.engine.valueextraction
+org.hibernate.validator.internal.util.privilegedactions=org.hibernate.validator.internal.util.privilegedactions
+org.hibernate.validator.internal=org.hibernate.validator.internal
```

Comment:
do we really need to list these packages for transform if they're not changing?

Reply:
It was the only way I could get the package versions to be changed. Is there a better way?

Reply:
Ah ok, that's fine then. It seems kind of clunky that the tool requires packages that won't be renamed to be defined in this file and the version mapping props file, but that's an issue with the xformer tool, not this PR so lets ignore it